osu.Game/Overlays/OverlayRulesetTabItem.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets;
+using osuTK.Graphics;
+using osuTK;
+using System;
+
+namespace osu.Game.Overlays
+{
+    public class OverlayRulesetTabItem : TabItem<RulesetInfo>, IHasAccentColour
+    {
+        protected readonly OsuSpriteText Text;
+        private readonly FillFlowContainer content;
+
+        public override bool PropagatePositionalInputSubTree => Enabled.Value && !Active.Value && base.PropagatePositionalInputSubTree;
+
+        private Color4 accentColour;
+
+        public Color4 AccentColour
+        {
+            get => accentColour;
+            set
+            {
+                if (accentColour == value)
+                    return;
+
+                accentColour = value;
+
+                updateState();
+            }
+        }
+
+        protected Action<Color4> OnStateUpdated;
```

Comment:
I'm not sure this should be a thing. Would rather have, say, a second bindable maybe?